tests/db_access/prepared_statements.cpp

```diff
@@ -600,10 +600,18 @@ go_bandit([](){
             AssertThat(varbinary.getStringView(), Equals("ij\0kl"s));
         });
 
-        it("can work with psReadValues (throws multiple rows error)", [&](){
+        it("can work with psReadValues (zero rows error)", [&](){
             int id{};
             BlobData blob{}, binary{}, varbinary{};
```

Comment:
I prefer made tests too much simple as possible - for instance, select varies of parameters has no added value for this test-case, the ORDERing the results in a query is useless as well.

This test-case may be reduced to form like this:
```suggestion
            int id{};
            AssertThrows(UnexpectedRowCountError,
                psReadValues("SELECT `id` FROM `test_superior_sqlpp`.`binary_data` LIMIT 0", connection, id)
            );
```

The same for `multiple rows error` test case below.